website/docs/best-practices/clone-incremental-models.md

```diff
@@ -11,21 +11,21 @@ Before you begin, you must be aware of a few conditions:
 - This strategy only works for warehouse that support zero copy cloning (otherwise `dbt clone` will just create pointer views).
 - Some teams may want to test that their incremental models run in both incremental mode and full-refresh mode.
 
-Imagine you've created a [Slim CI job](/docs/deploy/continuous-integration) in dbt Cloud and it is configured to: 
+Imagine you've created a [Slim CI job](/docs/deploy/continuous-integration) in <Constant name="cloud" /> and it is configured to: 
 
 - Defer to your production environment.
 - Run the command `dbt build --select state:modified+` to run and test all of the models you've modified and their downstream dependencies.
 - Trigger whenever a developer on your team opens a PR against the main branch.
 
 <Lightbox src="/img/best-practices/slim-ci-job.png" width="70%" title="Example of a slim CI job with the above configurations" />
 
-Now imagine your dbt project looks something like this in the DAG:
+Now imagine your <Constant name="dbt" /> project looks something like this in the DAG:
```

Comment:
Maybe this is better suited for a separate task but would this be a good opportunity to remove excessive use of `dbt`? A lot of pages establish the context without use having to refer to it as a "dbt project" each and every time